### PR TITLE
release to sonatype using GH Actions

### PR DESCRIPTION
**Pull request by octonato**
https://github.com/akka/akka-meta/issues/149

TODO
- [x] port integration tests
- [x] configure cron job
- [x] configure to run on release branch
- [ ] configure email notification
- [ ] configure required checks in settings page

### CODE REVIEW

**Comment by octonato** [comment]
We won't see the immediate effect of this. The workflow only runs in master, so after merging we should get a snapshot published and gustav update.

**Comment by octonato** [comment]
Failed because of Jabba

**Comment by patriknw** [comment]
I pushed 7649be6c03457ebe9e6e59e8d6407b88d1c6c711 to fix the tags and add the sonatypeProfileName

**Review by raboof — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        env:
+          GUSTAV_KEY: ${{ secrets.GUSTAV_KEY }}
+          GUSTAV_PASSPHRASE: ${{ secrets.GUSTAV_PASSPHRASE }}
+        run: |+
+          eval "$(ssh-agent -s)"
+          echo $GUSTAV_KEY | base64 -di > .github/id_rsa
+          chmod 600 .github/id_rsa
+          ssh-keygen -p -P "$GUSTAV_PASSPHRASE" -N "" -f .github/id_rsa
+          ssh-add .github/id_rsa
+          sbt "++2.13.4 docs/publishRsync"
```
Comment (ignasi35) [inline .github/workflows/release.yml:60]: Other workflows on this PR use 2.13.5 (e.g. https://github.com/akka/akka-management/pull/889/files#diff-cd0effce06f425599e96952f4d5f684641a683f46d9909e7c74a9f8b2f03ccd3R23).

**Review by ignasi35 — APPROVED** [review]
LGTM with a very tiny nitpick.

**Comment by octonato** [comment]
@raboof / @patriknw, thanks for pushing this PR over the line. 

I changed the settings for the branches. Not all options were checked.

**Pull request merged**